.github/workflows/cla.yml

```diff
@@ -19,5 +19,5 @@ jobs:
         with:
           path-to-signatures: 'signatures/cla.json'
           path-to-cla-document: 'https://github.com/exadel-inc/face-recognition-service/blob/develop/CLA.md'
-          branch: 'develop'
+          branch: 'CLA-signatures'
           allowlist: Elijas,daffy-,sbvalijon,stashess,kemerava,mmisachenka,isobolev,endypanda,mjakowiec,dmitry-boiko,pospielov,andreigaevsky,ivandodz,aabramovich-dev,DmitryRomaniuk,omorgunova-pl,github-actions
```

Comment:
Is it a good idea to remove all people from this list? So they have to approve CLA using proper procedure?

Reply:
I thought this procedure for outside Exadel contributors only, but as we have rapidly changing staff, it's a good idea to remove all.

Reply:
no, this procedure for all contributors, as we use personal github accounts